### PR TITLE
[zh-cn] Part4:updated the term "群集" to "集群"

### PR DESCRIPTION
**Pull request by dapengJacky**
Signed-off-by: jacky <peng.chen@daocloud.io>

issue:https://github.com/kubernetes/website/issues/33735
fix issue part4 file typo

### CODE REVIEW

**Comment by Sea-n** [comment]
The files are not fully synced with upstream (English) version.

Feel free to `/unhold` once it's fully checked and synced.
/hold

**Comment by dapengJacky** [comment]
@Sea-n 

> The files are not fully synced with upstream (English) version.
I know what u mean,so what should I do now,can u give some advice?

**Comment by ydFu** [comment]
> @Sea-n 
> 
> 
> 
> > The files are not fully synced with upstream (English) version.
> 
> 
> 
> I know what u mean,so what should I do now,can u give some advice?
@dapengJacky 


Please refer to this link for command line instructions for synchronization in both Chinese and English.




https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87

**Comment by dapengJacky** [comment]
> > @Sea-n
> > > The files are not fully synced with upstream (English) version.
> > 
> > 
> > I know what u mean,so what should I do now,can u give some advice?
> 
> @dapengJacky
> 
> Please refer to this link for command line instructions for synchronization in both Chinese and English.
> 
> https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87

I have synced with upstream (English) version,please review again,thanks

**Comment by tengqm** [comment]
please squash your PR into a single commit.

File: content/zh-cn/case-studies/zalando/index.html [hunk 1]
```diff
@@ -2,131 +2,195 @@
...
+
+<!-- 
+<p>Not to mention a unique opportunity for an infrastructure specialist like Jacobs. Just after he joined, the company began rewriting all their applications in-house. "That was generally our strategy," he says. "For example, we started with our own logistics warehouses but at first you don't know how to do logistics software, so you have some vendor software. And then we replaced it with our own because with off-the-shelf software you're not competitive. You need to optimize these processes based on your specific business needs."</p>
+-->
+<p>能拥有像 Jacobs 这样的基础设施专家的机会是非常独特的。就在他加入公司后，公司开始在内部重写他们的所有应用。“这通常是我们的策略，”他表示。“例如，我们从我们自己的物流仓库开始，但起初不知道如何做物流软件，所以得用一些供应商软件。然后我们用自己的软件替换了它，因为使用现成的软件是没有竞争力的。需要根据特定业务需求优化这些流程。”</p>
+
+<!-- 
+<p>In parallel to rewriting their applications, Zalando had set a goal of expanding beyond basic e-commerce to a platform offering multi-tenancy, a dramatic increase in assortments and styles, same-day delivery and even <a href="https://www.zalon.de">your own personal online stylist</a>.</p>
+-->
+<p>在重写其应用程序的同时，Zalando 设定了一个目标，即从基本电子商务扩展到提供多租户的平台、大量增加的分类和样式、当天送达，甚至<a href="https://www.zalon.de">您自己的平台个人在线造型师</a>。</p>
```
Comment (Sea-n) [inline content/zh-cn/case-studies/zalando/index.html:97]:
```suggestion
<p>在重写其应用程序的同时，Zalando 设定了一个目标，即从基本电子商务扩展到提供多租户的平台、大量增加的分类和样式、当天送达，甚至<a href="https://www.zalon.de">你自己的平台个人在线造型师</a>。</p>
```

File: content/zh-cn/case-studies/zalando/index.html [hunk 2]
```diff
@@ -2,131 +2,195 @@
...
+
+
+<!--
+<p>Jacobs's team also worked to improve the Kubernetes-AWS integration. "Thus you're very restricted. You need infrastructure to scale each autonomous team's idea." Plus, "there are still a lot of best practices missing," says Jacobs. The team, for example, recently solved a pod security policy issue. "There was already a concept in Kubernetes but it wasn't documented, so it was kind of tricky," he says. The large Kubernetes community was a big help to resolve the issue. To help other companies start down the same path, Jacobs compiled his team's learnings in a document called <a href="http://kubernetes-on-aws.readthedocs.io/en/latest/admin-guide/kubernetes-in-production.html">Running Kubernetes in Production</a>.</p>
+-->
+<p>Jacobs 的团队还致力于改进Kubernetes-AWS 集成。“由于许多限制条件,需要通过基础设施来扩展每个自主团队的想法。”此外，“仍然缺少很多最佳实践，”Jacobs说。例如，该团队最近解决了 pod 安全策略问题。“在Kubernetes已经有一个概念，但没有记录，所以有点棘手，”他说。大型的Kubernetes社区是解决这个问题的一大帮助。为了帮助其他公司走上同样的道路，Jacobs在一份名为<a href="http://kubernetes-on-aws.readthedocs.io/en/latest/admin-guide/kubernetes-in-production.html">在生产中运行 Kubernetes </a>的文件中汇编了他的团队的经验教训。</p>
```
Comment (Sea-n) [inline content/zh-cn/case-studies/zalando/index.html:166]:
```suggestion
<p>Jacobs 的团队还致力于改进 Kubernetes-AWS 集成。“由于许多限制条件，需要通过基础设施来扩展每个自主团队的想法。”此外，“仍然缺少很多最佳实践，”Jacobs 说。例如，该团队最近解决了 pod 安全策略问题。“在 Kubernetes 已经有一个概念，但没有记录，所以有点棘手，”他说。大型的Kubernetes社区是解决这个问题的一大帮助。为了帮助其他公司走上同样的道路，Jacobs 在一份名为<a href="http://kubernetes-on-aws.readthedocs.io/en/latest/admin-guide/kubernetes-in-production.html">在生产中运行 Kubernetes</a> 的文件中汇编了他的团队的经验教训。</p>
```

File: content/zh-cn/case-studies/zalando/index.html [hunk 3]
```diff
@@ -2,131 +2,195 @@
...
+-->
+<p>最后，Kubernetes 使 Zalando 能够引进和维护公司为发展其平台而设想的新产品。Jacobs 说：“时尚咨询产品使用 Scala，而我们以前的基础设施也难以实现这一点。”这是一个解决方法，该团队需要平台团队提供越来越多的支持，只是因为他们使用了不同的技术。现在有了Kubernetes，它就自主了。无论工作负载是什么，该团队都可以走自己的路，而 Kubernetes 可以防止其他瓶颈。</p>
+
+<!-- 
+<p>Looking ahead, Jacobs sees Zalando's new infrastructure as a great enabler for other things the company has in the works, from its new logistics software, to a platform feature connecting brands, to products dreamed up by data scientists. "One vision is if you watch the next James Bond movie and see the suit he's wearing, you should be able to automatically order it, and have it delivered to you within an hour," says Jacobs. "It's about connecting the full fashion sphere. This is definitely not possible if you have a bottleneck with everyone running in the same data center and thus very restricted. You need infrastructure to scale each autonomous team's idea."</p>
+-->
+<p>展望未来，Jacobs 将 Zalando 的新基础设施视为公司在进行的其他工程中的巨大推动因素，从新的物流软件到连接品牌的平台功能，以及数据科学家梦寐以求的产品。Jacobs说：“一个愿景是，如果你看下一部 James Bond 的电影，看看他穿的西装，你就应该能够自动订购，并在一小时内把它送到你身边。”“这是关于连接整个时尚领域。如果您遇到瓶颈，因为每个人都在同一个数据中心运行，因此限制很大，则这绝对是不可能的。需要基础设施来扩展每个自主团队的想法。”</p>
```
Comment (Sea-n) [inline content/zh-cn/case-studies/zalando/index.html:186]:
```suggestion
<p>展望未来，Jacobs 将 Zalando 的新基础设施视为公司在进行的其他工程中的巨大推动因素，从新的物流软件到连接品牌的平台功能，以及数据科学家梦寐以求的产品。Jacobs 说：“一个愿景是，如果你看下一部 James Bond 的电影，看看他穿的西装，你就应该能够自动订购，并在一小时内把它送到你身边。”“这是关于连接整个时尚领域。如果你遇到瓶颈，因为每个人都在同一个数据中心运行，因此限制很大，则这绝对是不可能的。需要基础设施来扩展每个自主团队的想法。”</p>
```

File: content/zh-cn/case-studies/zalando/index.html [hunk 4]
```diff
@@ -2,131 +2,195 @@
...
+-->
+<p>公司最初决定推迟编排，但随着团队迁移到 AWS，“我们看到团队在 AWS 上的基础设施和使用云资源方面遇到了难题，” Jacobs 说。</p>
+
+<!-- 
+<p>Zalandos 200+ autonomous engineering teams decide what technologies to use and could operate their own applications using their own AWS accounts. This setup proved to be a compliance challenge. Even with strict rules-of-play and automated compliance checks in place, engineering teams and IT-compliance were overburdened addressing compliance issues. "Violations appear for non-compliant behavior, which we detect when scanning the cloud infrastructure," says Jacobs. "Everything is possible and nothing enforced, so you have to live with violations (and resolve them) instead of preventing the error in the first place. This means overhead for teams—and overhead for compliance and operations. It also takes time to spin up new EC2 instances on AWS, which affects our deployment velocity."</p>
+-->
+<p>Zalando 200多人的自主工程团队研究使用哪些技术，并可以使用自己的 AWS 账户操作自己的应用程序。事实证明，此设置是一项合规性挑战。即使制定了严格的游戏规则和自动化的合规性检查，工程团队和 IT 合规性在解决合规性问题方面也负担过重。Jacobs 说："违规行为会出现，我们在扫描云基础架构时会检测到这些行为。“一切皆有可能，没有强制实施，因此您必须忍受违规行为（并解决它们），而不是一开始就防止错误。这些都会增加团队的开销，以及合规性和操作的开销。在 AWS 上启动新的 EC2 实例也需要时间，这会影响我们的部署速度。”</p>
```
Comment (Sea-n) [inline content/zh-cn/case-studies/zalando/index.html:126]:
```suggestion
<p>Zalando 200 多人的自主工程团队研究使用哪些技术，并可以使用自己的 AWS 账户操作自己的应用程序。事实证明，此设置是一项合规性挑战。即使制定了严格的游戏规则和自动化的合规性检查，工程团队和 IT 合规性在解决合规性问题方面也负担过重。Jacobs 说："违规行为会出现，我们在扫描云基础架构时会检测到这些行为。“一切皆有可能，没有强制实施，因此你必须忍受违规行为（并解决它们），而不是一开始就防止错误。这些都会增加团队的开销，以及合规性和操作的开销。在 AWS 上启动新的 EC2 实例也需要时间，这会影响我们的部署速度。”</p>
```

File: content/zh-cn/case-studies/zalando/index.html [hunk 5]
```diff
@@ -2,131 +2,195 @@
...
+{{< /case-studies/quote >}}
+
+<!-- 
+<p>At the moment, Zalando is running an initial 40 Kubernetes clusters with plans to scale for the foreseeable future. Once Zalando began migrating applications to Kubernetes, the results were immediate. "Kubernetes is a cornerstone for our seamless end-to-end developer experience. We are able to ship ideas to production using a single consistent and declarative API," says Jacobs. "The self-healing infrastructure provides a frictionless experience with higher-level abstractions built upon low-level best practices. We envision all Zalando delivery teams will run their containerized applications on a state-of-the-art reliable and scalable cluster infrastructure provided by Kubernetes."</p>
+-->
+<p>目前，Zalando正在运行最初的40个Kubernetes集群，并计划在可预见的将来进行扩展。一旦Zalando开始将申请迁移到Kubernetes，效果立竿见影。“ Kubernetes 是我们无缝端到端开发人员体验的基石。我们能够使用单一一致且声明性的 API 将创意运送到生产中，” Jacobs 说。“自愈基础架构提供了无摩擦体验，基于低级最佳实践构建了更高级别的抽象。我们设想所有 Zalando 交付团队都将在 Kubernetes 提供的最先进的可靠和可扩展集群基础架构上运行其容器化应用程序。”</p>
```
Comment (Sea-n) [inline content/zh-cn/case-studies/zalando/index.html:150]:
```suggestion
<p>目前，Zalando 正在运行最初的 40 个 Kubernetes 集群，并计划在可预见的将来进行扩展。一旦 Zalando 开始将申请迁移到 Kubernetes，效果立竿见影。“Kubernetes 是我们无缝端到端开发人员体验的基石。我们能够使用单一一致且声明性的 API 将创意运送到生产中，” Jacobs 说。“自愈基础架构提供了无摩擦体验，基于低级最佳实践构建了更高级别的抽象。我们设想所有 Zalando 交付团队都将在 Kubernetes 提供的最先进的可靠和可扩展集群基础架构上运行其容器化应用程序。”</p>
```

File: content/zh-cn/case-studies/zalando/index.html [hunk 6]
```diff
@@ -2,131 +2,195 @@
...
-</div>
-</section>
-<div class="banner3">
-  <div class="banner3text">
-    <!-- "This move to the cloud was necessary because in the data center you couldn’t have autonomous teams. You have the same infrastructure and it was very homogeneous, so you could only run your Java or Python app." -->
+
+<!-- 
+<p>With the old infrastructure "it was difficult to properly embrace new technologies, and DevOps teams were considered to be a bottleneck," says Jacobs. "Now, with this cloud infrastructure, they have this packaging format, which can contain anything that runs on the Linux kernel. This makes a lot of people pretty happy. The engineers love autonomy."</p>
+-->
+<p>Jacobs 说，由于旧基础设施“很难正确采用新技术，而 DevOps 团队被视为瓶颈。”“现在，有了这个云基础架构，它们有了这种打包格式，可以包含任何在Linux内核上运行的东西。这使得很多人相当高兴。工程师喜欢自主。”</p>
```
Comment (Sea-n) [inline content/zh-cn/case-studies/zalando/index.html:59]:
```suggestion
<p>Jacobs 说，由于旧基础设施“很难正确采用新技术，而 DevOps 团队被视为瓶颈。”“现在，有了这个云基础架构，它们有了这种打包格式，可以包含任何在 Linux 内核上运行的东西。这使得很多人相当高兴。工程师喜欢自主。”</p>
```

**Review by Sea-n — COMMENTED** [review]
It would be great to revise some sentence to be more fluent.

**Comment by dapengJacky** [comment]
> It would be great to revise some sentence to be more fluent.

thanks,I have fixed this

**Comment by Sea-n** [comment]
Ah, forgot to unhold since the content is synced.
/unhold

And overall looks good to me, wait for others to provide more suggestions.

**Comment by dapengJacky** [comment]
/cc @ydFu 
Could u help to review,thanks

File: content/zh-cn/case-studies/ygrene/index.html [hunk 7]
```diff
@@ -1,130 +1,187 @@
...
+  我们必须改变一些实践和代码，以及构建的方式，但我们能够在一个月左右的时间内将我们的主要系统安装到 Kubernetes 上，然后在两个月内投入生产。这对于金融公司来说是非常快的。
+
+new_case_study_styles: true
+heading_background: /images/case-studies/ygrene/banner1.jpg
+heading_title_logo: /images/ygrene_logo.png
+subheading: >
+  Ygrene: 使用原生云为金融行业带来安全性和可扩展性
+case_study_details:
+  - 公司: Ygrene
+  - 地点: 佩塔卢马，加利福尼亚州
```
Comment (ydFu) [inline content/zh-cn/case-studies/ygrene/index.html:19]:
```suggestion
  - 地点: Petaluma, Calif.
```
Location does not require translation.

File: content/zh-cn/case-studies/zalando/index.html [hunk 8]
```diff
@@ -2,131 +2,195 @@
...
-css: /css/style_zalando.css
+
+new_case_study_styles: true
+heading_background: /images/case-studies/zalando/banner1.jpg
+heading_title_logo: /images/zalando_logo.png
+subheading: >
+  欧洲领先的在线时尚平台通过云原生获得突破性进展
+case_study_details:
+  - 公司: Zalando
+  - 位置: 柏林, 德国
```
Comment (ydFu) [inline content/zh-cn/case-studies/zalando/index.html:13]:
```suggestion
  - 位置: Berlin, Germany
```

File: content/zh-cn/case-studies/zalando/index.html [hunk 9]
```diff
@@ -2,131 +2,195 @@
...
+
+<!-- 
+<p>"It started as a PHP e-commerce site which was easy to get started with, but was not scaling with the business' needs" says Jacobs, Head of Developer Productivity at Zalando.</p>
+-->
+<p>Zalando 的开发人员生产力主管 Jacobs 说：“它最初是一个 PHP 电子商务网站，很容易上手，但无法随业务需求扩展。”</p>
+
+<!-- 
+<p>At that time, the company began expanding beyond its German origins into other European markets. Fast-forward to today and Zalando now has more than 14,000 employees, 3.6 billion Euro in revenue for 2016 and operates across 15 countries. "With growth in all dimensions, and constant scaling, it has been a once-in-a-lifetime experience," he says.</p>
+-->
+<p>当时，公司开始从德国以外扩展到其他欧洲市场。快进到今天，Zalando 现在拥有超过 14000 名员工，2016 年收入为 36 亿欧元，业务遍及 15 个国家/地区。他表示：“这种全面快速的增长和不断扩展，一生只能有一次。”</p>
```
Comment (ydFu) [inline content/zh-cn/case-studies/zalando/index.html:87]:
```suggestion
<p>当时，公司开始从 Germany 以外扩展到其他欧洲市场。快进到今天，Zalando 现在拥有超过 14000 名员工，2016 年收入为 36 亿欧元，业务遍及 15 个国家/地区。他表示：“这种全面快速的增长和不断扩展，一生只能有一次。”</p>
```

File: content/zh-cn/case-studies/zalando/index.html [hunk 10]
```diff
@@ -2,131 +2,195 @@
...
+-->
+<p>Jacobs 的团队还致力于改进 Kubernetes-AWS 集成。“由于许多限制条件，需要通过基础设施来扩展每个自主团队的想法。”此外，“仍然缺少很多最佳实践，” Jacobs 说。例如，该团队最近解决了 pod 安全策略问题。“在 Kubernetes 已经有一个概念，但没有记录，所以有点棘手，”他说。大型的 Kubernetes 社区是解决这个问题的一大帮助。为了帮助其他公司走上同样的道路，Jacobs 在一份名为<a href="http://kubernetes-on-aws.readthedocs.io/en/latest/admin-guide/kubernetes-in-production.html">在生产中运行 Kubernetes </a> 的文件中汇编了他的团队的经验教训。</p>
+
+<!-- 
+{{< case-studies/quote >}}
+"The Kubernetes API allows us to run applications in a cloud provider-agnostic way, which gives us the freedom to revisit IaaS providers in the coming years... We expect the Kubernetes API to be the global standard for PaaS infrastructure and are excited about the continued journey."
+{{< /case-studies/quote >}}
+-->
+{{< case-studies/quote >}}
+“Kubernetes API 允许我们以与云提供商无关的方式运行应用程序，这使我们能够在未来几年中自由访问 IaaS 提供商...我们期望 Kubernetes API 成为 PaaS 基础设施的全球标准，并对未来的继续旅程感到兴奋。”
```
Comment (ydFu) [inline content/zh-cn/case-studies/zalando/index.html:174]:
```suggestion
“Kubernetes API 允许我们以与云提供商无关的方式运行应用程序，这使我们能够在未来几年中自由访问 IaaS 提供商...。 我们期望 Kubernetes API 成为 PaaS 基础设施的全球标准，并对未来的继续旅程感到兴奋。”
```

File: content/zh-cn/case-studies/zalando/index.html [hunk 11]
```diff
@@ -2,131 +2,195 @@
...
+-->
+<p>Zalando 是欧洲领先的在线时尚平台，自 2008 年成立以来，经历了指数级增长。2015 年，Zalando 计划进一步扩展其原有的电子商务站点，以扩展新的服务和产品，从而开始了<a href="https://jobs.zalando.com/tech/blog/radical-agility-study-notes/">彻底的变革</a>，因此形成了自主的自组织团队。这次扩展需要可以随工程组织的增长而扩展的基础架构。Zalando 的技术部门开始重写其应用程序，使其能够在云端运行，并开始将其基础架构从内部数据中心迁移到云。虽然编排没有立即被考虑，因为团队迁移到<a href="https://aws.amazon.com/">亚马逊网络服务</a>（AWS）：“我们体验过团队在 AWS 上进行基础设施架构和使用云资源时遇到的痛苦，”开发人员生产力主管 Henning Jacobs 说，“我们遇到了一些难题。对于团队和合规性来说，运营开销仍然过多。为了提供更好的支持，项目引入了集群管理。”</p>
```
Comment (ydFu) [inline content/zh-cn/case-studies/zalando/index.html:39]:
```suggestion
<p>Zalando 是欧洲领先的在线时尚平台，自 2008 年成立以来，经历了指数级增长。在 2015 年，Zalando 计划进一步扩展其原有的电子商务站点，以扩展新的服务和产品，从而开始了<a href="https://jobs.zalando.com/tech/blog/radical-agility-study-notes/">彻底的变革</a>，因此形成了自主的自组织团队。这次扩展需要可以随工程组织的增长而扩展的基础架构。Zalando 的技术部门开始重写其应用程序，使其能够在云端运行，并开始将其基础架构从内部数据中心迁移到云。虽然编排没有立即被考虑，因为团队迁移到<a href="https://aws.amazon.com/">亚马逊网络服务</a>（AWS）：“我们体验过团队在 AWS 上进行基础设施架构和使用云资源时遇到的痛苦，”开发人员生产力主管 Henning Jacobs 说，“我们遇到了一些难题。对于团队和合规性来说，运营开销仍然过多。为了提供更好的支持，项目引入了集群管理。”</p>
```

**Comment by dapengJacky** [comment]
@ydFu  thanks,I have fixed

**Review by ydFu — APPROVED** [review]
LGTM,It looks like this is your first PR,keep it up.

/lgtm

**Pull request merged**